core/core.rng

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<grammar ns="http://libero.pub" xmlns="http://relaxng.org/ns/structure/1.0"
+    datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+
+    <include href="attributes/lang.rng"/>
+
+</grammar>
```

Comment:
The `core/` folder provides standard building blocks for a schema (can be considered schemas of their own I guess, describing a small or large element rather than a whole document); it doesn't provide a document-level schema if not in the form of the `tests/` ones which can be used as examples by people defining their own. These nuances can be captured in the `README` as an entry point for people looking at this repository?

Reply:
Definitely, though let's do it in or at the same time as #1.